std/node/_fs/_fs_exists.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+type ExitsCallback = (exists: boolean) => void;
+
+/* Deprecated in node api */
+
+export function exists(path: string, callback: ExitsCallback): void {
```

Comment:
This was added in 47a580293eb5c176497264f1c3f108bf6b2c480d

Please merge with master.

Reply:
merged done 👍